src/clj/yummy/parser.clj

```diff
@@ -19,8 +19,8 @@
      (doseq [tag *tags*]
        (.registerTag ctor (name tag)))
      ctor)
-   (org.yaml.snakeyaml.representer.Representer.)
-   (org.yaml.snakeyaml.DumperOptions.)))
+   (org.yaml.snakeyaml.representer.Representer.
```

Comment:
`Representer` constructor takes in [DumperOptions](https://javadoc.io/doc/org.yaml/snakeyaml/latest/org/yaml/snakeyaml/representer/Representer.html) since `snakeyaml 2.0`